views/ManageUsers.html

```diff
@@ -68,6 +68,7 @@
     <br/>
 
 </div>
-<script>manageUsers.init()</script>
```

Comment:
One line would be a little cleaner or we could have a 
<script>
managerUsers.init()
</script> 

block